validator/validator-main.protoascii

```diff
@@ -2238,6 +2238,7 @@ tags: {
     mandatory: true
     value_url: {
       protocol: "data"
+      protocol: "http" # For parity with amp-img's mandatory-src-or-srcset.
       protocol: "https"
       allow_relative: true  # Will be set to false at a future date.
```

Comment:
Can you remove the comment on this line?

The reason we wanted to set these true was that relative might allow an URL to be http even if it wasn't explicitly mentioned as http. Now that we allow http, it's fine to allow relative.

Reply:
Done in 6d60cb73d.